trace-dispatcher/src/Cardano/Logging/Tracer/Composed.hs

```diff
@@ -119,3 +126,32 @@ mkCardanoTracer' trStdout trForward mbTrEkg name namesFor severityFor privacyFor
         case mbEkgTrace <> mbForwardTrace <> mbStdoutTrace of
           Nothing -> pure $ Trace NT.nullTracer
           Just tr -> pure (preFormatted backends tr)
+
+-- A simple dataPointTracer which supports building a namespace and entering a hook
+-- function.
+mkDataPointTracer :: forall dp. ToJSON dp
+  => Trace IO DataPoint
+  -> (dp -> [Text])
+  -> IO (Trace IO dp)
+mkDataPointTracer trDataPoint namesFor = do
+    let tr = NT.contramap DataPoint trDataPoint
+    pure $ withNamesAppended namesFor tr
+
+documentTracer ::
+     TraceConfig
+  -> Trace IO a
+  -> Documented a
+  -> IO [(Namespace, DocuResult)]
+documentTracer trConfig trace trDoc = do
+    res <- catch (do
+              configureTracers trConfig trDoc [trace]
+              pure True)
+          (\(e :: SomeException) -> do
```

Comment:
This function contains a strange alignment, it should be fixed.

Reply:
Done